codeforboston/maple
# Mobile add edit testimony

# Summary

Close #1306 

Implemented the mobile view of Add/Edit Testimony pages according to this Figma: https://www.figma.com/file/NPlnWH6AiI2IQMebd8ardO/Mobile-Screens?type=design&node-id=558%3A8200&mode=design&t=k0B4pQcwXIj2PFa8-1.
Made the "View Policy" tab stick to the top in mobile view.
Added the "Previous" button to the "Select Your Legislators" page. 

# Checklist


# Screenshots
![screencapture-localhost-3000-submit-testimony-2023-11-21-17_38_28](https://github.com/codeforboston/maple/assets/39149005/00a134da-6a6d-421a-b50c-569538e3f4cf)
![screencapture-localhost-3000-submit-testimony-2023-11-21-17_39_20](https://github.com/codeforboston/maple/assets/39149005/41999842-68e2-407c-b8a8-d59a30babf02)
![screencapture-localhost-3000-submit-testimony-2023-11-21-17_40_08](https://github.com/codeforboston/maple/assets/39149005/a32b00f4-bacf-4339-90b1-ce90df2dedc2)
![screencapture-localhost-3000-submit-testimony-2023-11-21-17_41_09](https://github.com/codeforboston/maple/assets/39149005/53e5f062-e2d8-4742-a737-c1a1cd49876e)
![screencapture-localhost-3000-submit-testimony-2023-11-21-17_42_07](https://github.com/codeforboston/maple/assets/39149005/264f01a4-7b66-43f4-b8b5-f0c555b38b63)
![screencapture-localhost-3000-submit-testimony-2023-11-21-17_44_46](https://github.com/codeforboston/maple/assets/39149005/5d8ab8f7-7298-4bd6-ab14-f8021af10634)
![image](https://github.com/codeforboston/maple/assets/39149005/d323ebe2-8f3c-4697-96fe-375a5a60aa80)


# Steps to test/reproduce

1. Go any bill.
2. Click "Create Testimony" on the right side of the page.
3. Follow the instructions to complete your testimony to see the changes on those pages
